src/Form/PhpAgentSettingsForm.php

```diff
@@ -135,6 +135,16 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $config->get('phpAgent.captureThrowable'),
     ];
 
+    $form['privacy'] = [
+      '#type' => 'details',
+      '#title' => $this->t('Privacy settings'),
+    ];
+    $form['privacy']['send_user'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Send user details'),
+      '#default_value' => $config->get('phpAgent.privacy.send_user'),
+    ];
```

Comment:
**CHANGE REQUEST**

"Separate section" meant to be a separate settings page/tab, not just separate section in the agent config.
The reasoning is that this is not really a setting of the agent, it configures which details to log like the other tab we have to configure which pages to log for. There is no setting in the PHP agent for that.

Reply:
@krystalcode I have added privacy to a separate tab. Could you please review